deposit/src/main/java/edu/unc/lib/deposit/CleanupDepositJob.java

```diff
@@ -155,35 +153,23 @@ private void deleteStagedFiles(Model m) {
 			ni.close();
 		}
 	}
-	
+
 	// Cleanup files and directories specifically requested be cleaned up by an earlier job
 	private void deleteCleanupFiles(Model m) {
 		List<String> cleanupPaths = new ArrayList<>();
-		
+
 		// Create a list of files that need to be cleaned up
 		NodeIterator it = m.listObjectsOfProperty(CdrDeposit.cleanupLocation);
 		try {
 			while (it.hasNext()) {
 				RDFNode n = it.nextNode();
 				URI cleanupUri = URI.create(n.asLiteral().getString());
-				
-				SharedStagingArea area = getStorageArea(cleanupUri);
-				if (area == null) {
-					continue;
-				}
-				
-				URI storageUri = null;
-				try {
-					storageUri = area.getStorageURI(cleanupUri);
-				} catch (StagingException e) {
-					LOG.error("Could not resolve storage URI: {}", cleanupUri.toString(), e);
-				}
-				
-				CleanupPolicy p = area.getIngestCleanupPolicy();
-				switch (p) {
+
+				CleanupPolicy cleanupPolicy = stagingManager.getCleanupPolicy(cleanupUri);
+				switch (cleanupPolicy) {
 				case DELETE_INGESTED_FILES:
```

Comment:
Is this case simply intended to fall through to the DELETE_INGESTED_FILES_EMPTY_FOLDERS case?